chainercv/links/model/yolo/yolo_v2.py

```diff
@@ -150,7 +159,12 @@ def __init__(self, n_fg_class=None, pretrained_model=None):
         self._default_bbox = np.array(default_bbox, dtype=np.float32)
 
         if path:
-            chainer.serializers.load_npz(path, self, strict=False)
+            if use_pretrained_class_weights:
+                chainer.serializers.load_npz(path, self, strict=False)
+            else:
+                utils.link.load_npz_with_ignore_names(
+                    path, self, strict=False,
+                    ignore_names=['subnet/W', 'subnet/b'])
```

Comment:
`subnet` contains the weights for localization and objectness. This code removes them.